docs/src/main/asciidoc/_admin-guide/General_configuration_concepts.adoc

```diff
@@ -42,8 +42,8 @@ The most significant part of the configuration in `domain.xml` and
 `standalone.xml` is the configuration of one (in `standalone.xml`) or
 more (in `domain.xml`) "profiles". A profile is a named set of subsystem
 configurations. A subsystem is an added set of capabilities added to the
-core server by an extension (see "Extensions" above). A subsystem
-provides servlet handling capabilities; a subsystem provides an Jakarta Enterprise Beans
+core server by an extension (see <<extensions,"Extensions">> above). A subsystem
+provides servlet handling capabilities; a subsystem provides a Jakarta Enterprise Beans
```

Comment:
Let's fix this across the code base - I did open https://github.com/wildfly/wildfly/pull/18042.